src/server/svr_jobfunc.c

```diff
@@ -1373,7 +1373,7 @@ svr_chkque(job *pjob, pbs_queue *pque, char *hostname, int mtype)
 
 	if (pque->qu_attr[QE_ATR_AclGroupEnabled].at_val.at_long)
 		if (acl_check(&pque->qu_attr[QE_ATR_AclGroup],
-			pjob->ji_wattr[(int)JOB_ATR_egroup].at_val.at_str,
+			pjob->ji_wattr[(int)JOB_ATR_euser].at_val.at_str,
```

Comment:
Is it your intention to check whether one of the user's supplemental GIDs matches one of the allowed GIDs? I'm guessing that is the case because you are passing the effective user ID as opposed to the effective group ID to acl_check(). I would expect to see calls to getpwnam() and getgrouplist() in gacl_match() if that is the case.